pom.xml

```diff
@@ -222,7 +222,7 @@
     <dependency>
       <groupId>org.testng</groupId>
       <artifactId>testng</artifactId>
-      <version>7.0.0</version>
+      <version>7.1.1</version>
```

Comment:
should be 7.1.0

Reply:
7.1.1 is the latest - it’s just hasn’t been pushed yet from bintray. Once it’s synced then we can murge this pr.